tests/e2e/cucumber/features/smoke/fullTextSearch.ocis.feature

```diff
@@ -82,6 +82,18 @@ Feature: Search
 
     When "Brian" enables the option to search in file content
     And "Brian" searches "Cheers" using the global search bar
+    Then following resources should be displayed in the files list for user "Brian"
+      | resource                      |
+      | textfile.txt                  |
+      | testFolder/innerTextfile.txt  |
+      | fileToShare.txt               |
+      | fileWithTag.txt               |
+      | withTag.txt                   |
+      | spaceFolder/spaceTextfile.txt |
+    When "Brian" opens the following file in texteditor
+      | resource                      |
+      | textfile.txt                  |
+    And "Brian" closes the file
```

Comment:
we already have this step. Won't it work?
```suggestion
    And "Brian" closes the editor
```

Reply:
Same step is used for closing "pdfViewer" and "mediaviewer". So, should I change them to the editor since the pdf Viewer and media viewer isn't an editor? 
There was a function that closes the editor but I separated that code to make the steps more sequential. Initially, it was "Then following resources should be displayed in the files" and then "Brian logs out". First, you close the editor then you log out.